src/uas/UASWaypointManager.cc

```diff
@@ -1073,16 +1088,20 @@ void UASWaypointManager::sendWaypoint(quint16 seq)
 
         mavlink_mission_item_int_t *wp;
 
-
         wp = waypoint_buffer.at(seq);
         wp->target_system = uasid;
         wp->target_component = MAV_COMP_ID_MISSIONPLANNER;
 
-        emit updateStatusString(QString("Sending waypoint ID %1 of %2 total").arg(wp->seq).arg(current_count));
-
+        if (current_state == WP_SENDLIST_SENDWPSINT) {
+            mavlink_msg_mission_item_int_encode(uas->getSystemId(), uas->getComponentId(), &message, wp);
+        } else if (current_state == WP_SENDLIST_SENDWPSFLOAT) {
+            mavlink_mission_item_t wp_float;
+            convertMavlinkMissionItem(wp, &wp_float);
+            mavlink_msg_mission_item_encode(uas->getSystemId(), uas->getComponentId(), &message, &wp_float);
+        }
+        else return;
```

Comment:
Perhaps we should print a logmessage (debug level) to show that something went wrong, instead of simply do nothing??

Reply:
Indeed, that's a good idea :)